src/components/annotations/AnnotationDetails.vue

```diff
@@ -246,7 +240,6 @@ export default {
     CytomineTags,
     CytomineProperties,
     AttachedFiles,
-    AnnotationCommentsModal,
```

Comment:
This line can be safely removed because the `AnnotationCommentsModal` is not used in the template, so it has not to be declared in the components section.

This modal is managed pro grammatically by Buefy (and `AnnotationCommentsModal` is correctly imported in this file).